api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -680,8 +681,19 @@ ml_single_get_tensors_info (ml_single_h single, gboolean is_input,
 
   ML_SINGLE_GET_VALID_HANDLE_LOCKED (single_h, single, 0);
 
-  ml_single_get_tensors_info_from_filter (single_h->filter, is_input, info);
+  /* allocate handle for tensors info */
+  status = ml_tensors_info_create (info);
+  if (status != ML_ERROR_NONE)
+    goto exit;
+
+  input_info = (ml_tensors_info_s *) (*info);
+
+  if (is_input)
+    ml_tensors_info_clone (input_info, &single_h->in_info);
```

Comment:
```ml_tensors_info_clone``` has a return value. Don't we need to check it?

Reply:
input_info is not initialized at this point. ```ml_tensors_info_clone ``` may generate seg-fault or return ML_ERROR_INVALID_PARAMETER (if input_info is NULL)

Reply:
If app developer's given parameter (info) is incorrect (NULL), ```ml_tensors_info_clone``` may return ERROR.

Reply:
We need to check it although most of errors of ```ml_tensors_info_clone``` are going to be prevented by checking the return of ```ml_tensors_info_create```.

Reply:
updated.